Makefile

```diff
@@ -12,7 +12,7 @@ NATIVE_EXT = .so
 DLLMAP_OS_NAME = linux
 endif
 
-XA_CONFIGURATION  = XAIntegrationDebug
+XA_CONFIGURATION  = Debug
```

Comment:
Do we need to keep this around?

I guess we should remove more stuff like `xa-fxcop` rule, `xa-all` rule, `XA_INTEGRATION_OUTPUTS` variable, ...